contracts/veLON.sol

```diff
@@ -285,8 +394,8 @@ contract veLON is IveLON, ERC721, Ownable, ReentrancyGuard {
 
     // Go over weeks to fill history and calculate what the current pool point is
     function _syncGlobalPoints(Point memory poolLastPoint, uint256 _epoch) private returns (Point memory, uint256) {
-        // block slope = dBlock/dTime
-        // If last point is already recorded in this block, slope=0
+        // block decliningRate = dBlock/dTime
+        // If last point is already recorded in this block, decliningRate=0
         // But that's ok because we know the block in such case
         uint256 blockSlope = 0;
```

Comment:
Should we change `blockSlope` to `blockDecliningRate`?

Reply:
Right, renamed as `blockRate` in 6bbc955